website/docs/cloud-docs/api-docs/state-versions.mdx

```diff
@@ -59,7 +59,7 @@ Creating state versions requires permission to read and write state versions for
 -> **Note:** This endpoint cannot be accessed with [organization tokens](/cloud-docs/users-teams-organizations/api-tokens#organization-api-tokens). You must access it with a [user token](/cloud-docs/users-teams-organizations/users#api-tokens) or [team token](/cloud-docs/users-teams-organizations/api-tokens#team-api-tokens).
```

Comment:
```suggestion
-> **Note:** You cannot access this endpoint with [organization tokens](/cloud-docs/users-teams-organizations/api-tokens#organization-api-tokens). You must access it with a [user token](/cloud-docs/users-teams-organizations/users#api-tokens) or [team token](/cloud-docs/users-teams-organizations/api-tokens#team-api-tokens).
```

Suggest changing "This endpoint cannot be accessed with..." to "You cannot access this endpoint with..." to avoid passive voice.